nav2_costmap_2d/plugins/costmap_filters/keepout_filter.cpp

```diff
@@ -74,6 +74,11 @@ void KeepoutFilter::initializeFilter(
     std::bind(&KeepoutFilter::filterInfoCallback, this, std::placeholders::_1));
 
   global_frame_ = layered_costmap_->getGlobalFrameID();
+
+  declareParameter("override_lethal_cost", rclcpp::ParameterValue(false));
+  node->get_parameter(name_ + "." + "override_lethal_cost", override_lethal_cost_);
+  declareParameter("lethal_override_cost", rclcpp::ParameterValue(252));
+  node->get_parameter(name_ + "." + "lethal_override_cost", lethal_override_cost_);
```

Comment:
We can probably add a clamping function so that values larger than `MAX_NON_OBSTACLE` can be handled:

```suggestion
  node->get_parameter(name_ + "." + "lethal_override_cost", lethal_override_cost_);
  
  // clamp lethal_override_cost_ in case if higher than MAX_NON_OBSTACLE is given
  lethal_override_cost_ = \
    std::clamp<unsigned int>(lethal_override_cost_, FREE_SPACE, MAX_NON_OBSTACLE);
```